api/rds.proto

```diff
@@ -300,7 +300,10 @@ message Route {
     RedirectAction redirect = 3;
   }
 
-  // See base.Metadata description.
+  // See base.Metadata description for message structure. The Metadata field
+  // can be used to provide the Router filter additional information
+  // about the Route. It can be used for configuration, stats, and logging.
+  // The filter name should be specified as "envoy.router".
```

Comment:
It should be envoy.router when destined for the router filter, but other filters might also consume route information.

Reply:
gotcha, updated